lib/versioning/poetry/readme.md

```diff
@@ -1,3 +1,3 @@
 Poetry versioning is a little like a mix of PEP440 and SemVer.
 
-Currently Renovate's implementation is based off npm versioning, but it is being migrated to be based off PEP440 to be more compatible with Poetry's behavior.
+Currently Renovate's implementation is based off npm versioning. This works by parsing versions using the same patterns and similar normalization rules as Poetry, passing them to the npm versioning implementation, and then reversing the normalizations. This allows Renovate to meaningfully compare the SemVer-style versions allowed in pyproject.toml to the PEP440 representations used on PyPI. These are equivalent for major.minor.patch releases, but different for pre-, post-, and dev releases.
```

Comment:
```suggestion
Currently Renovate's implementation is based off npm versioning.
This works by parsing versions using the same patterns and similar normalization rules as Poetry, passing them to the npm versioning implementation, and then reversing the normalizations.
This allows Renovate to meaningfully compare the SemVer-style versions allowed in `pyproject.toml` to the PEP440 representations used on PyPI.
These are equivalent for major.minor.patch releases, but different for pre-, post-, and dev releases.
```

Let's use one sentence per line.
I've added backticks so that `pyproject.toml` renders in a monospaced font.